addons/notes/src/__tests__/index.ts

```diff
@@ -1,5 +1,5 @@
 import addons from '@storybook/addons';
-import { withNotes } from '..';
+import { withNotes } from '../addon';
```

Comment:
this file should be renamed to index.test.ts as well. BTW, why rename `index` to `addon` at all?

Reply:
I did this when there was a index.ts and a react.ts 
I talked to Norbert and asked him what's the difference and that I find it kinda confusing for an outsider to really know what's going on

Now, we have just 1 file (react.ts was removed), this may not be necessary anymore but still: `index.ts` doesn't tell you anything about what this file does - let's discuss if you disagree 🙂

Reply:
I can't argue against If it helps to dive into the code=) I just don't like unneeded code movements, and I do like the consistency across the code base =)

Reply:
Maybe some people should talk about a naming convention before we touch every addon? I mean we can leave them as they are - it's just a good moment to do stuff like this in one refactoring process